CHANGELOG.md

```diff
@@ -1,7 +1,19 @@
 # Release History
 
-## 1.1.0-beta.3 (Unreleased)
+## 1.1.0-beta.4 (Unreleased)
```

Comment:
We don't want to bump this version. `.3` in the feature branch was correct. @weshaggard - what do you suggest we do here?

The workflow in this feature branch includes frequent pull/merge from master. Should we limit those to only prior to a beta release?

Reply:
Unfortunately there isn't any magic solution here you will just have to carefully watch the versions of the components that you plan to potentially ship in the feature branch and ensure the version matches that versioning plan. 

I would suggest you take pretty frequent changes as we don't want a big surprise late in the process if the merge has any issues.